2024-frontend/src/router/vue3.js

```diff
@@ -60,79 +60,79 @@ const vue3routes = [
   },
   {
     path: "/developpement-et-apis",
-    name: "DeveloperAPI",
-    component: DeveloperAPI,
+    name: "Developpeurs",
+    component: Developpeurs,
     meta: {
       title: "Développement et APIs",
       siteMap: action,
     },
   },
   {
     path: "/mentions-legales",
-    name: "LegalNotices",
-    component: LegalNotices,
+    name: "MentionsLegales",
+    component: MentionsLegales,
     meta: {
       title: "Mentions légales",
       siteMap: site,
     },
   },
   {
     path: "/accessibilite",
-    name: "AccessibilityDeclaration",
-    component: AccessibilityDeclaration,
+    name: "Accessibilite",
+    component: Accessibilite,
     meta: {
       title: "Déclaration d'accessibilité",
       siteMap: site,
     },
   },
   {
     path: "/cgu",
-    name: "CGU",
-    component: CGU,
+    name: "ConditionsGeneralesUtilisation",
+    component: ConditionsGeneralesUtilisation,
     meta: {
       title: "Conditions générales d'utilisation",
       siteMap: site,
     },
   },
   {
     path: "/contact",
-    name: "ContactPage",
-    component: ContactPage,
+    name: "Contact",
+    component: Contact,
     meta: {
       title: "Contactez-nous",
       siteMap: site,
     },
   },
   {
     path: "/importer-des-donnees/achats",
-    name: "ImportPurchases",
-    component: ImportPurchases,
+    name: "GestionnaireImportsAchat",
```

Comment:
```suggestion
    name: "GestionnaireImportAchat",
```
au singulier ?

Reply:
```suggestion
    name: "GestionnaireImportAchats",
```
voir le dernier mot au pluriel ?

Reply:
Aaah je bug trop sur ce nom de page, j'avais pensé à "Importer" mais ça ne suit pas notre logique ! 
Je crois que je préfère tout au singulier 🤔 Car on dit "quoi" et pas besoin d'ajouter la notion d'il y en a plusieurs ? En soit on pourrait importer 1 cantine

Reply:
Dans ce cas là le pluriel a du sens je trouve, car ces pages permettent justement d'importer 1 ou plusieurs cantines/achats/bilans.

Reply:
GestionnaireImportCantines
- 1 seul import
- plusieurs objets

Reply:
C'est goood convaincue 😁